console/HAL-1578_Customise_Browser_Title.adoc

```diff
@@ -0,0 +1,58 @@
+= Customise Browser Title
+:author:            Harald Pehl
+:email:             hpehl@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+Customise the title of the management console. A custom title can be specified in the user settings of the console. The title can contain the following placeholders:
+
+- `%n`: The `name` attribute of the the root resource
+- `%o`: The `organization` (standalone) resp. `domain-organization` attribute of the the root resource
+
+If the title is not specified, the built in title will be used. If a placeholder is used, but the attribute is undefined, the built in title will be used.
```

Comment:
Wouldn't it be better if users could directly benefit from the enhancement by using the name placeholder in default title value?

Reply:
AFAIK the name attribute is undefined for domain mode.

Reply:
It seems like it has a name, though not a nice one:
```
[domain@localhost:9990 /] :read-attribute(name=name)
{
    "outcome" => "success",
    "result" => "Unnamed Domain"
}
```

Reply:
@hpehl I understand there will be no `%n` in title by default? IMHO it would make sense...

Reply:
Ok, I agree. I'll change the implementation: Default title will be "%n | Management Console"

Reply:
Thanks!